### DISCUSSION TITLE
Docs for each Uno SDK version available on the site and kept in sync

### DISCUSSION THREAD

**Discussion by trydalch** [unoplatform/uno#18071] [Ideas]
It would be very helpful if the docs were versioned with the Uno SDK releases. At least for major versions. I'm not sure if that's something that docfx supports, but right now it seems there are a number of docs with instructions that don't work with Uno SDK 5 and the single project model, or at a minimum are confusing as they have language that seems to reference the multi-project model.

I've now run into it with both the MapControl (instructions don't work with the latest version of Uno), and WebAuthenticationBroker instructions and seen multi-project language on a number of other pages.

**Comment by morning4coffe-dev** [comment]
Hi @trydalch,

Thank you for your suggestion. While I don’t believe docfx currently supports this feature, we should definitely ensure the instructions reflect the latest versions. Would it be possible for you to create an issue and link the specific pages where you encountered problems?

Additionally, you mentioned issues with the MapControl and WebAuthenticationBroker—could you clarify which steps were problematic for you?

Thanks in advance for your help!
